pkgs/applications/networking/browsers/mozilla-plugins/mplayerplug-in/default.nix

```diff
@@ -18,7 +18,8 @@ stdenv.mkDerivation rec {
      else "");
 
   buildInputs = [ pkgconfig browser (browser.gtk) libXpm gettext ];
-  
+  propagatedUserEnvPkgs = [ mplayer ];
```

Comment:
IMHO, it's no safe solution to rely on the user environment to find `mplayer`. Instead, the path to mplayer should be hard-coded. If that is impossible for some reason, then there should be a wrapper script that sets `$PATH` up properly in for the context in which the plugin is run.